code/modules/power/cell.dm

```diff
@@ -17,14 +17,26 @@
 	var/rigged = 0		// true if rigged to explode
 	var/minor_fault = 0 //If not 100% reliable, it will build up faults.
 	var/chargerate = 100 //how much power is given every tick in a recharger
+	var/self_recharge = 0 //does it self recharge, over time, or not?

```

Comment:
Oh. Thanks. I didn't know that was a thing; that problem occurred to me when putting this in, but I didn't want to have non-charging cells in object processing, so I opted for this--didn't realize that there was a proc like this. Will implement.